add-solution-component/action.yml

```diff
@@ -43,6 +43,11 @@ inputs:
     description: 'Indicates whether other solution components that are required by the solution component that you are adding should also be added to the unmanaged solution'
     required: false
 
+  run-asynchronously:
+    description: 'Runs operation asynchronously.'
```

Comment:
This will also need an update to `src/actions/add-solution-component/index.ts` and `.../actionLaunch.ts`, as well as checking in the artifacts produced from `npm run update-dist`

Reply:
Fixed